modules/react/icon/lib/SystemIcon.tsx

```diff
@@ -55,7 +55,6 @@ export interface SystemIconProps
    * The size of the SystemIcon in `px`.
    */
   size?: number | string | undefined;
-  className?: string; //investigate more
```

Comment:
Come back and check the impact on this.
- [ ] it's ok

Reply:
There is no impact. `SystemIcon` uses `createComponent` and the default element is a `span`, so all props that a `span` element has, SystemIcon inherits including `className`.